File: 기초 프로그래밍 실습/250327/score.py
score = int(input("점수를 입력하세요 : "))

if score > 100:
    print("X")
elif score >= 100:
    print("A")
elif score >= 90:
    print("B")
elif score >= 80:
    print("C")
elif score >= 70:
    print("E")
else:
    print("F")

print("학점입니다.")

File: 프로그래밍 언어(1)/C/251029/score.c
#include <stdio.h>

#define STUDENT 3

int main() {
    int score[STUDENT];
    int total = 0;
    for (int i=0; i<STUDENT; i++) {
        scanf("%d", &score[i]);
        total += score[i];
    }
    int avg = total / STUDENT;
    printf("%d", avg);
    
    return 0;
}